tq/transfer.go

```diff
@@ -33,11 +33,21 @@ type Transfer struct {
 	Oid           string       `json:"oid,omitempty"`
 	Size          int64        `json:"size"`
 	Authenticated bool         `json:"authenticated,omitempty"`
-	Actions       ActionSet    `json:"actions,omitempty"`
+	Actions       mapActionSet `json:"actions,omitempty"`
+	Links         mapActionSet `json:"_links,omitempty"`
 	Error         *ObjectError `json:"error,omitempty"`
 	Path          string       `json:"path,omitempty"`
 }
 
+// HybridActions returns an action set composed of the "actions" property, as
+// well as the "_links" property. It is implemented by a multiActionSet, and
+// therefore retains that ordering.
+func (t *Transfer) HybridActions() ActionSet {
```

Comment:
What do you think about having a `Action()` function that checks `Actions` and `Links`? I'm not sure `multiActionSet` is necessary. Basically: `t.Actions.Get(string)` => `t.Action(string)`. I think the wrapped types and interface are a bit much for this.